src/Agent.Worker/TaskRunner.cs

```diff
@@ -576,14 +576,15 @@ private void PublishTelemetry(Definition taskDefinition, HandlerData handlerData
             ArgUtil.NotNull(taskDefinition.Data, nameof(taskDefinition.Data));
 
             var useNode10 = AgentKnobs.UseNode10.GetValue(ExecutionContext).AsString();
+            var expectedExecutionHandler = (taskDefinition.Data.Execution != null && taskDefinition.Data.Execution.All != null) ? string.Join(", ", taskDefinition.Data.Execution.All) : "";
```

Comment:
It could be a little simplified:
`taskDefinition.Data.Execution?.All != null`
instead of
`taskDefinition.Data.Execution != null && taskDefinition.Data.Execution.All != null`

Reply:
Makes sence, thanks